modules/xmpp/SDPUtil.js

```diff
@@ -612,57 +603,69 @@ const SDPUtil = {
      * types are also stripped. If the resulting mline would have no codecs,
      * it's disabled.
      *
-     * @param {object} videoMLine the video mline object from an sdp as parsed
-     * by transform.parse.
+     * @param {object} mLine the mline object from an sdp as parsed by transform.parse.
      * @param {string} codecName the name of the codec which will be stripped.
      */
-    stripVideoCodec(videoMLine, codecName) {
-        if (!videoMLine || !codecName) {
+    stripCodec(mLine, codecName, highProfile = false) {
```

Comment:
The `highProfile` argument seems to be very h264 codec specific. It's also missing JSDoc description. Would it make sense to pass an item matcher function instead? In this example it would be `item => item.config.includes('profile-level-id=64')`